mdbenchmark/mdengines/gromacs.py

```diff
@@ -101,5 +104,16 @@ def analyze_run(sim):
     else:
         module = sim.categories['version']
 
+    # We were unable to find the information in the log file or the log file
+    # does not exist (yet). Append to the correct list in the provided
+    # dictionary.
+    if not ns_day or not ncores:
+        if not bool(ns_day):
```

Comment:
Why not `if ns_day == 0` or `if np.allclose(ns_day, 0)`?

Reply:
I changed `ns_day` and `ncores` to equal `False` on [lines 87-88](https://github.com/bio-phys/MDBenchmark/pull/27/files/9ae51acb59bbac121427b2fc3eb143acab1dfff8#diff-3a3de48ca1e8f5d2cc919ebde57cef2dR87-R88).

I could actually change it back again to equal `0` as a default, just thought `if not ns_day` is nicer in reading than `if ns_day == 0`.

Reply:
Actually, you can just use `np.nan` as a default and also return that value in case we cannot read any information. This is sufficient to mark the error. Then there is no need for additional error handling in this function. Your error handling idea of using `nan` in the analysis function is pretty good and by returning nan we make better use of it.